lib/index.js

```diff
@@ -34,6 +34,8 @@ var CoverallsReporter = function(rootConfig, helper, logger) {
         filepath = path.resolve(rootConfig.basePath, reporter.dir);
         break;
       }
+    } else {
+     throw new Error("coverage reporter should precede coveralls");
```

Comment:
I guess that could work. I was following my wrong usage of it, which was:

```javascript
coverageReporter: {
 dir: 'coverage/',
 reporters: ['lcov', 'html']
}
```

What if the user is using another `type`? The plugin would still try to find the file, wouldnt it?